ouroboros-consensus-test/test-consensus/Test/Consensus/BlockchainTime/Simple.hs

```diff
@@ -293,24 +294,23 @@ testOverrideDelay :: forall m. (IOLike m, MonadTime m, MonadDelay (OverrideDelay
                   -> Int  -- ^ Number of slots to collect
                   -> OverrideDelay m [SlotNo]
 testOverrideDelay systemStart slotLength maxClockRewind numSlots = do
-    result <- withRegistry $ \registry -> do
+    withRegistry $ \registry -> do
       time <- simpleBlockchainTime
```

Comment:
Check whether `simpleBlockchainTime` could be `withBlockchainTime` etc.

Reply:
I did change this to `bracketWithPrivateRegistry`. It's a bit odd when there's no close function for the `btime`, but it's still better capturing the intent.